test/controllers/active_storage/blobs_controller_test.rb

```diff
@@ -65,8 +63,6 @@ class ActiveStorage::BlobsControllerTest < ActionDispatch::IntegrationTest
   describe 'when the user is not authenticated' do
     before { subject }
 
-    it 'redirects and flashes a message' do
```

Comment:
To fix, though I don't think this actually worked before, but was ignored due to missing translations.